react/ui-components/src/hoc/app-config-screen-builder/AppLocalisationWrapper.js

```
@@ -0,0 +1,153 @@
+
+const locReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case "SET_LOC":
+      return action.payload.localisationData;
+    case "default":
+      return state;
+    case "ADD_MISSING_KEY":
+      if (state.find((item) => item.code === action.payload.code)) {
+        return state;
+      } else {
+        const newEntry = action.payload.enabledModules.reduce((acc, locale) => ({ ...acc, [locale.value]: "" }), {
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)